### DISCUSSION TITLE
What's the difference between Domain list and User define Rules

### DISCUSSION THREAD

**Discussion by sharego** [tarampampam/random-user-agent#243] [Q&A] [answered]
Is Domain list can not use `*` and `?`， and User define Rules can use those specials match characters？

They both using for black model or white model？

**Comment by tarampampam** [comment, marked as answer]
> Is Domain list can not use * and ?， and User define Rules can use those specials match characters？

Yes

> They both using for black model or white model？

Depending on the selected mode
